gatsby-config.js

```diff
@@ -1,4 +1,20 @@
 const path = require(`path`)
+const targetJournal = 31;
+
+const findSearchData = (node, entry) => {
+
+  let data = ""
+  if(node.publication) {
+    for(var i = 0; i < node.publication.journals.length; i++) {
```

Comment:
`var`

Reply:
The ``var`` removal is done.  Material-UI seems to need to know what all values are for the autocomplete, which I don't know if I have easy access to, at this point.  I'd request that autocomplete not hold this up

Reply:
👍